Src/Support/Google.Apis.Auth/Google.Apis.Auth.csproj

```diff
@@ -0,0 +1,72 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- nupkg information -->
+  <Import Project="..\CommonPackageInfo.xml" />
+  <PropertyGroup>
+    <Title>Google APIs Client Library</Title>
+    <PackageTags>Google</PackageTags>
+    <Description>
+The Google APIs Client Library is a runtime client for working with Google services.
```

Comment:
Yes. This multiline description is moved into the nupack, and I've verified it's correct in the final nupkg using NuGetPackageExplorer